Model/GetItemsToDeductFromOrder.php

```diff
@@ -52,16 +52,12 @@ public function execute(Order $order): array
     {
         $itemsToOrder = [];
 
-        /** @var \Magento\Sales\Model\Order\Item $orderItem */
+        /** @var \Magento\Sales\Model\Order\Item|null $orderItem */
         foreach ($order->getAllVisibleItems() as $orderItem) {
-            if ($orderItem->getParentItem() !== null) {
-                continue;
-            }
-            // This code was added as quick fix for merge mainline
```

Comment:
Umm, I'd leave this code comment in, maybe amend it so we know that it's referring to `null === $orderItem` and not `$orderItem->getParentItem()`